### PR TITLE
Add trailing newline to files without one

### LINKED ISSUE #145: Comment on the last line of a FSH file causes an error
If the last line of a FSH file is a comment or if the last line ends with an inline comment, the following error is reported: 

`error: extraneous input '//' expecting {<EOF>, KW_ALIAS, KW_PROFILE, KW_EXTENSION, KW_INSTANCE, KW_INVARIANT, KW_VALUESET, KW_CODESYSTEM}`

This may be related to the fix for #130 because I did not see this issue prior the the PR that addressed that bug.

Example FSH files that will report that error:

```
Instance: Example
InstanceOf: Patient
// Comment
```

```
Instance: Example
InstanceOf: Patient // Comment
```

### PR DESCRIPTION
**Pull request by kjmahalingam**
Fixes #145 

There was an issue with comments (line and block) causing a syntax error if they are at the end of a file, and there is no newline in the file. This PR addresses the issue by solving it the way some other languages do; adding a trailing newline to the end of the file content being processed if there isn't one already.

I'd like someone on Windows to test this out, since I know newlines can be kind of messy across operating systems.

I'm going to add a couple of tests to verify these work as well; stand by for that.

### CODE REVIEW

**Comment by kjmahalingam** [comment]
Tests added!

**Comment by kjmahalingam** [comment]
Ah, that's a good call. I didn't think about that.
I will make those changes.

**Comment by kjmahalingam** [comment]
All feedback has been addressed, @cmoesel . Let me know if there's anything else that needs to be changed.

If @ngfreiter or @mint-thompson is able to test this manually, as well, to make sure newlines don't cause issues on a Windows OS, that would be great!

**Comment by ngfreiter** [comment]
I gave this a test run and did not see any issues with it working with Windows 👍 .

File: src/import/FSHImporter.ts [hunk 1]
```diff
@@ -111,6 +111,7 @@ export class FSHImporter extends FSHVisitor {
       docs.push(doc);
 
       // Create and store context for main import process
+      if (!rawFSH.content.endsWith('\n')) rawFSH.content += '\n';
```
Comment (cmoesel) [inline src/import/FSHImporter.ts]: I'd really like to see a comment here indicating _why_ we're tacking on the newline.  A few years into the future, this line may look really suspicious to someone.

Comment (cmoesel) [reply src/import/FSHImporter.ts]: Also, we could avoid removing the `readonly` from `RawFSH` by changing this to:
```ts
const content = rawFSH.content.endsWith('\n') ? rawFSH.content + '\n';
const ctx = this.parseDoc(content, rawFSH.path);
```
This also means we're not mutating the original input.

File: src/import/RawFSH.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 export class RawFSH {
   public readonly path: string;
-  constructor(public readonly content: string, path?: string) {
+  constructor(public content: string, path?: string) {
```
Comment (cmoesel) [inline src/import/RawFSH.ts]: You can now revert this change so it is `readonly` again.
